ai_worker/tests/src/test_case_isolation.py

```
@@ -0,0 +1,455 @@
+Given: 여러 케이스의 데이터가 저장됨
+When: 케이스별 조회/삭제/검증 수행
+Then: 케이스 간 데이터 격리 보장
+"""
+
+import unittest
+import tempfile
+import shutil
+from pathlib import Path
+from datetime import datetime
+
+from src.storage.metadata_store import MetadataStore
```

Pylint (4.0.3)

[error] No name 'storage' in module 'src'

(E0611)